openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -570,7 +570,8 @@ def get_all_courses(cls, orgs=None, filter_=None):
         # Note: If a newly created course is not returned in this QueryList,
         # make sure the "publish" signal was emitted when the course was
         # created. For tests using CourseFactory, use emit_signals=True.
-        course_overviews = CourseOverview.objects.all()
+        num_courses = getattr(settings, 'ITEMS_VISIBLE_HOMEPAGE', 12)
```

Comment:
Hi @huynguyen-groove ,
Can you please update this varibable ITEMS_VISIBLE_HOMEPAGE name to `COURSE_ITEMS_ON_LMS_HOMEPAGE`